content/broker/broker-transition/how-to-get-started/_index.en.md

```diff
@@ -0,0 +1,32 @@
+---
+title: How To Get Started
+linktitle: How To Get Started
+description: How to get started using the Broker Transition solution from Altinn 2 to Altinn 3   
+tags: [architecture, solution]
+toc: false
+weight: 1
+---
+
+## ServiceOwners
+In order to use the Broker Transition solution in Altinn 2 to create, upload and retrieve file metadata, a service owner must complete the following steps.
+1. Have an existing Altinn 2 Broker Service.
+2. Have or create a corresponding Altinn 3 Broker Resource.
```

Comment:
Add "Se how to get started with Altinn Broker here: TODO: Add link to Get Started"